lib/block_view.go

```diff
@@ -9473,78 +9769,77 @@ func (bav *UtxoView) _flushBalanceEntriesToDbWithTxn(txn *badger.Txn) error {
 }
 
 func (bav *UtxoView) FlushToDbWithTxn(txn *badger.Txn) error {
-	// Flush the utxos to the db.
-	if err := bav._flushUtxosToDbWithTxn(txn); err != nil {
-		return err
+	// Only flush to BadgerDB if Postgres is disabled
+	if bav.Postgres == nil {
+		if err := bav._flushUtxosToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushProfileEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushPKIDEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushPostEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushLikeEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushFollowEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushDiamondEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushMessageEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
+		if err := bav._flushBalanceEntriesToDbWithTxn(txn); err != nil {
+			return err
+		}
 	}
 
+	// Always flush to BadgerDB.
```

Comment:
So the stuff below this line is what's yet to be moved right?

Reply:
Is there anything hard about moving the stuff below? Also, separately, is there anything more to do other than migrating these over to postgres.go?

Reply:
Nothing hard, just saving the easy stuff for last